File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    let maxRow = 0;
    let maxColumn = 0;
    
    for (let [row, column] of sizes) {
        const [big, small] = row > column ? [row, column] : [column, row];
        maxRow = Math.max(big, maxRow);
        maxColumn = Math.max(small, maxColumn);
        
        }
    return maxRow * maxColumn;
    }
    

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    max_x = 0
    max_y = 0
    for size in sizes:
        x = max(size)
        y = min(size)
        if x > max_x:
            max_x = x
        if y > max_y:
            max_y = y
    return max_x * max_y